malfeasance/wire/malfeasance.go

```diff
@@ -324,6 +325,12 @@ type InvalidPrevATXProof struct {
 
 func (p *InvalidPrevATXProof) isProof() {}
 
+type DoubleMarryProof struct {
+	// TODO: implement
+}
+
+func (p *DoubleMarryProof) isProof() {}
+
```

Comment:
This belongs in the `activation` package.

Reply:
I added the general process for this to https://github.com/spacemeshos/go-spacemesh/pull/6043